### PR TITLE
Replace plain text with attributes for content in the platform module

### PR DESCRIPTION
**Pull request by michellemacrh**
This PR is specific to the 2.3 branch and version of our docs. The main PR is #1119

Replace plain-text with attributes where appropriate

https://issues.redhat.com/browse/AAP-19894

### CODE REVIEW

File: downstream/assemblies/navigator/assembly-executing-content.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ ifdef::context[:parent-context: {context}]
 
 
 [role="_abstract"]
-Now that you have your {ExecEnvName} built, you can use automation content navigator to validate that the content will be run in the same manner as the automation controller will run it.
+Now that you have your {ExecEnvName} built, you can use automation content navigator to validate that the content will be run in the same manner as the {ControllerName} will run it.
```
Comment (ianf77) [inline downstream/assemblies/navigator/assembly-executing-content.adoc:13]: content will be run -> content is run

File: downstream/modules/catalog/proc-preparing-worker-installer.adoc [hunk 2]
```diff
@@ -105,13 +105,13 @@ You can now proceed to run the installer.
 # ./setup.sh
 -----
 
-Running the automation services catalog worker will create an application and an application token, install the necessary packages, connect to cloud.redhat.com, and start the service.
+Running the {CatalogName} worker will create an application and an application token, install the necessary packages, connect to cloud.redhat.com, and start the service.
```
Comment (ianf77) [inline downstream/modules/catalog/proc-preparing-worker-installer.adoc:108]: will create -> creates

File: downstream/modules/platform/con-about-automation-mesh.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 [id="con-automation-mesh"]
 
-= About automation mesh
+= About {MeshName}
 
 [role="_abstract"]
-Automation mesh is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
+{AutomationMeshStart} is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
 
 {PlatformName} 2 replaces Ansible Tower and isolated nodes with {ControllerName} and {HubName}. {ControllerNameStart} provides the control plane for automation through its UI, Restful API, RBAC, workflows and CI/CD integration, while Automation Mesh can be used for setting up, discovering, changing or modifying the nodes that form the control and execution layers.
```
Comment (ianf77) [inline downstream/modules/platform/con-about-automation-mesh.adoc:9]: There's an Automation Mesh here. Attribute?

File: downstream/modules/platform/ref-platform-notes.adoc [hunk 4]
```diff
@@ -32,7 +32,7 @@ For more details, see: https://access.redhat.com/solutions/1519083
 
 *  The RADIUS protocol uses `md5` to encrypt passwords in `Access-Request` queries; link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/ent_auth.html#radius[RADIUS Authentication] is not supported for systems where FIPS mode is enabled.
 
-.Notes for Red Hat Enterprise Linux setups
+.Notes for {RHEL} setups
 
 * PackageKit can frequently interfere with the installation/update mechanism. Consider disabling or removing PackageKit if installed prior to running the setup process.
```
Comment (ianf77) [inline downstream/modules/platform/ref-platform-notes.adoc:37]: prior -> before

File: downstream/titles/catalog/getting-started/master.adoc [hunk 5]
```diff
@@ -12,7 +12,7 @@ include::aap-common/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 == About {CatalogName}
 
-{CatalogNameStart} is a service within the Red Hat Ansible Automation Platform.
+{CatalogNameStart} is a service within the {PlatformName}.
 {CatalogNameStart} allows customers to organize and govern product catalog sources on Ansible Tower across various environments.
```
Comment (ianf77) [inline downstream/titles/catalog/getting-started/master.adoc:16]: allows -> enables

**Review by ianf77 — COMMENTED** [review]
A few comments

File: downstream/modules/platform/con-about-automation-mesh.adoc (hunk 3 above)
Comment (michellemacrh) [reply downstream/modules/platform/con-about-automation-mesh.adoc:9]: Ah, I it didn't get captured as it's Automation Mesh as opposed to automation mesh. I'll double check the code with that in mind.

**Review by ianf77 — APPROVED** [review]

**Pull request merged**